android/README.md

```diff
@@ -78,16 +78,17 @@ Building these projects follow the same steps as KMAPro:
 2. Run `./build.sh`
 
 Keyman Engine for Android library (**keyman-engine.aar**) is now ready to be imported in any project.
-A copy can be found at **android/Samples/KMSample1/app/libs/keyman-engine.aar**.
 
 ## How to Use Keyman Engine for Android Library
 
-1. Copy **keyman-engine.aar** into **[Your project folder]/app/libs/** folder.
+1. Add **keyman-engine.aar** into **[Your project folder]/app/libs/** folder.
+    a. If you built the Keyman Engine, get the copy from **android/Samples/KMSample1/app/libs/keyman-engine.aar**
```

Comment:
We need to clarify regarding alpha vs beta vs stable quality. Recommendation is to use a stable release (when it is available) from the download server.

Reply:
OK, I've added the preference for downloading the latest stable KM engine